### PR TITLE
helpers nit: mark classes with only statics `sealed`

### PR DESCRIPTION
**Pull request by kevmoo**
Fix some relative imports

### CODE REVIEW

File: lib/src/helpers/enums.dart [hunk 1]
```diff
@@ -5,9 +5,9 @@
 /// Helper layer library that exposes enums commonly used from `dart:html`.
 library;
 
-import '../../web.dart';
+import '../dom/html.dart';
 
-abstract final class KeyLocation {
```
Comment (srujzs) [inline lib/src/helpers/enums.dart:10]: Why `sealed` over `abstract final`? These don't have constructors and we don't really need exhaustiveness (although I suppose `abstract sealed` doesn't hurt).

File: lib/src/helpers/events/providers.dart [hunk 2]
```diff
@@ -5,12 +5,12 @@
...
 import 'dart:js_interop_unsafe';
 
-import '../../../helpers.dart' show Device;
 import '../../../web.dart';
+import '../enums.dart' show Device;
 import 'streams.dart';
 
 /// Statically accessible `EventStreamProvider`s for all event types.
-class EventStreamProviders {
+sealed class EventStreamProviders {
```
Comment (srujzs) [inline lib/src/helpers/events/providers.dart:13]: Could also make this `abstract sealed`.

Comment (kevmoo) [reply lib/src/helpers/events/providers.dart:13]: `abstract sealed` just adds the ability to extend...which...why?

https://dart.dev/language/modifier-reference

Comment (srujzs) [reply lib/src/helpers/events/providers.dart]: Turns out `abstract sealed` doesn't even exist. :) `abstract final` it is then.

**Review by srujzs — APPROVED** [review]

**Pull request merged**